ChatApp/Custom+UI/ChatView.swift

```diff
@@ -0,0 +1,102 @@
+//
+//  ChatView.swift
+//  ChatApp
+//
+//  Created by bakhva  on 29.04.23.
+//
+
+import UIKit
+
+final class ChatView: UIView {
+    
+    // MARK: Components
+    lazy var textFieldContainer: TextFieldContainerView = {
+        let container = TextFieldContainerView()
+        container.translatesAutoresizingMaskIntoConstraints = false
+        container.textField.font = .systemFont(ofSize: ChatViewConstants.textFieldFont)
+        return container
+    }()
+    
+    lazy var chatTableView: UITableView = {
+        let tableView = UITableView()
+        tableView.backgroundColor = .clear
+        tableView.translatesAutoresizingMaskIntoConstraints = false
+        tableView.separatorStyle = .none
+        tableView.allowsSelection = false
+        tableView.register(SenderTableViewCell.self, forCellReuseIdentifier: SenderTableViewCell.identifier)
+        tableView.register(ReceiverTableViewCell.self, forCellReuseIdentifier: ReceiverTableViewCell.identifier)
+
+        return tableView
+    }()
+    
+    // MARK: Init
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        addSubviews()
+        addConstraints()
+    }
+    
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
```

Comment:
let add this, its better
```suggestion
    
    @available(*, unavailable)
    required init?(coder: NSCoder) {
        fatalError("init(coder:) has not been implemented")
    }
```